packages/kbn-apm-config-loader/src/config_loader.ts

```diff
@@ -24,5 +27,19 @@ export const loadConfiguration = (
   const configPaths = getConfigurationFilePaths(argv);
   const rawConfiguration = getConfigFromFiles(configPaths);
   applyConfigOverrides(rawConfiguration, argv);
-  return new ApmConfiguration(rootDir, rawConfiguration, isDistributable);
+
+  apmConfig = new ApmConfiguration(rootDir, rawConfiguration, isDistributable);
+  return apmConfig;
+};
+
+export const getConfiguration = (serviceName: string): ApmAgentConfig => {
+  // integration test runner starts a kibana server that import the module without initializing APM.
+  // so we need to check initialization of the config.
+  // note that we can't just load the configuration during this module's import
+  // because jest IT are ran with `--config path-to-jest-config.js` which conflicts with the CLI's `config` arg
+  // causing the config loader to try to load the jest js config as yaml and throws.
+  if (apmConfig) {
+    return apmConfig.getConfig(serviceName);
+  }
+  return {};
```

Comment:
let's return `undefined` or null to distinguish the case when it's not defined from the case when it's empty. 
The downstream code already handles it as `undefined` https://github.com/elastic/kibana/blob/331c6f55a4746dcc5db653c501a2f477486619ab/src/core/server/http_resources/get_apm_config.ts#L14